client/src/Components/HomePage/HomePage.js

```diff
@@ -1,12 +1,19 @@
-import logo from "../../logo.svg";
 import "./HomePage.css";
 import LandingPage from "../LandingPage/LandingPage";
-
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
+import Navbar from "../navBar/header";
+import CreatePost from "../CreatePost/CreatePost";
```

Comment:
use snake case while naming your folders and files

Reply:
@jasce sir, I made all changes as per you guidance, how to update it?

Reply:
have you renamed the directories ?